### PR TITLE
new version of chronos and fixing local predictions by name and ID for datetimes

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

**Comment by mmerce** [comment]
Tests pass now. Please, review again.

File: bigml/util.py [hunk 1]
```diff
@@ -609,11 +609,12 @@ def check_no_training_missings(input_data, fields, weight_field=None,
 
     """
     for field_id, field in fields.items():
-        if (not field_id in input_data and \
-            field['summary']['missing_count'] == 0 and \
+        if (field["optype"] != "datetime" and \
+                (not field_id in input_data and \
```
Comment (unmonoqueteclea) [inline bigml/util.py]: Bad indentation... And is this parenthesis, before the `not`, needed? It's hard to understand all the conditions.

**Comment by mmerce** [comment]
@unmonoqueteclea, you should now have write access, so you can proceed to merge if that's OK.

File: bigml/util.py (hunk 1 above)
Comment (mmerce) [reply bigml/util.py]: Let me change that, but see the comment below.

Comment (mmerce) [reply bigml/util.py]: In fact, I think that's not a bad indentation. When if is multiline, there's a double indentation to differentiate the multiple lines in the condition from the first line of the if body.

Comment (unmonoqueteclea) [reply bigml/util.py]: Sorry, I didn't know it! Thank you for the info!

**Pull request merged**